bin/pygrb/pycbc_pygrb_grb_info_table

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python
+
+# Copyright (C) 2021 Francesco Pannarale & Michael Patel
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+# =============================================================================
+# Preamble
+# =============================================================================
+from __future__ import division
+
+import sys
+import logging
+import argparse
+import pycbc.version
+import pycbc.results
+import lal
+import numpy
+from datetime import datetime
+from pycbc.detector import Detector
+from pycbc.results.pygrb_postprocessing_utils import get_antenna_dist_factor
+
+__author__ = "Francesco Pannarale <francesco.pannarale@ligo.org>"
+__version__ = pycbc.version.git_verbose_msg
+__date__ = pycbc.version.date
+__program__ = "pycbc_pygrb_grb_info_table"
+
+# =============================================================================
+# Main script starts here
+# =============================================================================
+description = 'Create GRB info table.'
+usage = __program__ + ' [--options]'
+
+parser = argparse.ArgumentParser(usage=usage, description=description,
```

Comment:
@ahnitz, @spxiwh, before I follow up on this, can you confirm that it is indeed the convention we want?  I ask because I poked around and saw conflicting approaches within PyCBC.  I would rather make a single swipe :-)

Reply:
`__doc__` is now used in all these post-processing scripts and `usage` is gone.